dev/tasks/tasks.yml

```diff
@@ -452,16 +452,18 @@ tasks:
 
 {############################## Wheel Linux ##################################}
 
-{% for ci, arch, arch_alias, x_y, manylinux in [("github", "amd64", "x86_64", "2_17", "2014"),
-                                                ("travis", "arm64", "aarch64", "2_17", "2014")] %}
-  wheel-manylinux{{ manylinux }}-{{ python_tag }}-{{ arch }}:
+{% for ci, arch, arch_alias, x_y, manylinux_version, manylinux in [("github", "amd64", "x86_64", "2_17", "2014", "manylinux2014"),
+                                                                   ("github", "amd64", "x86_64", "2_28", "2-28", "manylinux_2_28"),
+                                                                   ("travis", "arm64", "aarch64", "2_17", "2014", "manylinux2014"),
+                                                                   ("travis", "arm64", "aarch64", "2_28", "2-28", "manylinux_2_28")] %}
+  wheel-{{ manylinux }}-{{ python_tag }}-{{ arch }}:
     ci: "{{ ci }}"
     template: python-wheels/{{ ci }}.linux.{{ arch }}.yml
     params:
       python_version: "{{ python_version }}"
-      manylinux_version: "{{ manylinux }}"
+      manylinux_version: "{{ manylinux_version }}"
     artifacts:
-      - pyarrow-{no_rc_version}-{{ python_tag }}-{{ abi_tag }}-manylinux_{{ x_y }}_{{ arch_alias }}.manylinux{{ manylinux }}_{{ arch_alias }}.whl
+      - pyarrow-{no_rc_version}-{{ python_tag }}-{{ abi_tag }}-manylinux_{{ x_y }}_{{ arch_alias }}.{{ manylinux }}_{{ arch_alias }}.whl
```

Comment:
it seems the new artifacts generated are not exactly matching this pattern:
https://github.com/ursacomputing/crossbow/actions/runs/3984737889/jobs/6831289469
We look for: `pyarrow-11.0.0.dev500-cp37-cp37m-manylinux_2_28_x86_64.manylinux_2_28_x86_64.whl ` but the generated one on the url: https://github.com/ursacomputing/crossbow/releases/download/actions-ca19472285-github-wheel-manylinux_2_28-cp37-amd64/pyarrow-11.0.0.dev500-cp37-cp37m-manylinux_2_28_x86_64.whl
```
pyarrow-11.0.0.dev500-cp37-cp37m-manylinux_2_28_x86_64.whl
pyarrow-11.0.0.dev500-cp37-cp37m-manylinux_2_28_x86_64.manylinux_2_28_x86_64.whl
```
The manylinux_2014 ones:
`pyarrow-11.0.0.dev496-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl` seems correct https://github.com/ursacomputing/crossbow/actions/runs/3978737424/jobs/6820692950#step:18:27

Reply:
> it seems the new artifacts generated are not exactly matching this pattern: https://github.com/ursacomputing/crossbow/actions/runs/3984737889/jobs/6831289469 We look for: `pyarrow-11.0.0.dev500-cp37-cp37m-manylinux_2_28_x86_64.manylinux_2_28_x86_64.whl ` but the generated one on the url: https://github.com/ursacomputing/crossbow/releases/download/actions-ca19472285-github-wheel-manylinux_2_28-cp37-amd64/pyarrow-11.0.0.dev500-cp37-cp37m-manylinux_2_28_x86_64.whl
> 
> ```
> pyarrow-11.0.0.dev500-cp37-cp37m-manylinux_2_28_x86_64.whl
> pyarrow-11.0.0.dev500-cp37-cp37m-manylinux_2_28_x86_64.manylinux_2_28_x86_64.whl
> ```
> 
> The manylinux_2014 ones: `pyarrow-11.0.0.dev496-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl` seems correct https://github.com/ursacomputing/crossbow/actions/runs/3978737424/jobs/6820692950#step:18:27

I think there are some variables in `ci/scripts/python_wheel_manylinux_build.sh` that aren't taking the variable changes into the account. I'll take a look at this.

Reply:
https://github.com/apache/arrow/pull/33805/commits/3924533a9b4c4c61a74ad0ae1b0df3f713a27bef handles the different artifact schemas produced by audiwheel with a jinja if-else statement.